ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -604,18 +608,27 @@ def get_database_names(self) -> Iterable[str]:
     def get_view_definition(
```

Comment:
should the method still be named `get_view_definition`? maybe `get_schema_definition` or something more fitting to the new use case would be better

Reply:
Changed the method name `get_view_definition ` ->  `get_schema_definition`